frontend/src/concepts/pipelines/content/tables/experiment/ExperimentTableRow.tsx

```diff
@@ -24,11 +24,20 @@ const ExperimentTableRow: React.FC<ExperimentTableRowProps> = ({
 }) => {
   const { namespace } = usePipelinesAPI();
 
+  const isArchived = window.location.href.includes('/archived');
```

Comment:
I think we typically identify where we are location using react router helpers, and this could be identified using useLocation or useParams. I'm also thinking this row could identify this via a prop passed down from the table as well.